src/test/java/uk/gov/hmcts/reform/sscs/service/HearingsServiceTest.java

```diff
@@ -0,0 +1,150 @@
+package uk.gov.hmcts.reform.sscs.service;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+import uk.gov.hmcts.reform.sscs.exception.UnhandleableEventTypeException;
+import uk.gov.hmcts.reform.sscs.exception.UnhandleableHearingStateException;
+import uk.gov.hmcts.reform.sscs.exception.UpdateCaseException;
+import uk.gov.hmcts.reform.sscs.model.HearingEvent;
+import uk.gov.hmcts.reform.sscs.model.HearingState;
+import uk.gov.hmcts.reform.sscs.model.HearingWrapper;
+import uk.gov.hmcts.reform.sscs.model.single.hearing.HearingResponse;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.assertj.core.api.Assertions.assertThatNoException;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static uk.gov.hmcts.reform.sscs.model.HearingState.CREATE_HEARING;
+
+class HearingsServiceTest {
+
+
+    private static final long HEARING_REQUEST_ID = 12345;
+    private static final String HMC_STATUS = "TestStatus";
+    private static final long VERSION = 1;
+    private static final String CANCELLATION_REASON_CODE = "TestCancelCode";
+    private static final long CASE_ID = 1625080769409918L;
+    private static final long MISSING_CASE_ID = 99250807409918L;
+
+    @Mock
+    private CcdCaseService ccdCaseService;
+
+    private HearingsService hearingsService;
+    private HearingWrapper wrapper;
+    private SscsCaseDetails expectedCaseDetails;
+
+    @BeforeEach
+    void setup() {
+        MockitoAnnotations.openMocks(this);
```

Comment:
You can use the Mockito extension and @InjectMocks to avoid needing this

Reply:
That didn't work, so I changed it to @ExtendWith(MockitoExtension.class)